observer/probers/tls/tls.go

```diff
@@ -123,7 +123,7 @@ func (p TLSProbe) probeExpired(timeout time.Duration) bool {
 	}
 
 	root := peers[len(peers)-1].Issuer
-	if root.Organization[0] != p.rootOrg || root.CommonName != p.rootCN {
+	if p.rootCN != "" && (root.Organization[0] != p.rootOrg || root.CommonName != p.rootCN) {
```

Comment:
This logic is duplicated with probeUnexpired below. Since the logic is growing in complexity, now would be a good time to extract it into its own function to ensure it stays consistent.

Reply:
The only thing I can think of to avoid someone omitting the root check unintentionally is to omit the check only if both the rootCN _and_ rootOrg are empty (rather than just rootCN). Hopefully it will also be at least slightly intuitive that we can't check that the root is the one we expect unless we tell it what to expect.